#include<iostream>
using namespace std;

void accept_record( int &number ){
	cout << "Number	:	";
	cin >> number;
}
void print_record( int &result ){
	cout << "Result	:	" << result << endl;
}
int main( void ){
	cout <<"File/Connection Open"<<endl;

	int num1;
	accept_record( num1 );

	int num2;
	accept_record( num2 );

	try{
		if( num2 == 0 ){
			double ex = 0.0;
			throw ex;
		}else{
			int result = num1 / num2;
			print_record( result );
		}
	}catch( int ex ){
		cout << "handling exception in int catch block "<<endl;
	}

	cout <<"File/Connection Close"<<endl;
	return 0;
}
